File: python/update_opentypeextension.py
import http.client

conn = http.client.HTTPSConnection("graph.microsoft.com")

payload = "{ }"

headers = { 'content-type': "application/json" }

conn.request("PATCH", "/v1.0/groups(%2737df2ff0-0de0-4c33-8aee-75289364aef6%27)/threads(%27AAQkADJizZJpEWwqDHsEpV_KA==%27)/posts(%27AAMkADJiUg96QZUkA-ICwMubAADDEd7UAAA=%27)/extensions(%27Microsoft.OutlookServices.OpenTypeExtension.Com.Contoso.Estimate%27)", payload, headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))

File: node/update_opentypeextension.js
var http = require("https");

var options = {
  "method": "PATCH",
  "hostname": "graph.microsoft.com",
  "port": null,
  "path": "/v1.0/groups(%2737df2ff0-0de0-4c33-8aee-75289364aef6%27)/threads(%27AAQkADJizZJpEWwqDHsEpV_KA==%27)/posts(%27AAMkADJiUg96QZUkA-ICwMubAADDEd7UAAA=%27)/extensions(%27Microsoft.OutlookServices.OpenTypeExtension.Com.Contoso.Estimate%27)",
  "headers": {
    "content-type": "application/json"
  }
};

var req = http.request(options, function (res) {
  var chunks = [];

  res.on("data", function (chunk) {
    chunks.push(chunk);
  });

  res.on("end", function () {
    var body = Buffer.concat(chunks);
    console.log(body.toString());
  });
});

req.write("{ }");
req.end();
